config/initializers/cookie_rotator.rb

```diff
@@ -0,0 +1,21 @@
+# config/initializers/cookie_rotator.rb
+Rails.application.config.after_initialize do
+    Rails.application.config.action_dispatch.cookies_rotations.tap do |cookies|
```

Comment:
Vielleicht kann da @rnestler kurz draufschauen, der hatte das doch glaub mal erwähnt :)

Reply:
Gute Idee, wir wollen den Branch aber heute mergen, von dem her wird das für ihn wahrscheinlich schwierig.

Reply:
Das einzige was mir auffällt ist, dass die Einrückung 4 anstatt 2 spaces ist. Sonst ist es exakt wie in der Dokumentation :)